src/Grids/regular_cartesian_grid.jl

```diff
@@ -205,7 +205,7 @@ end
 short_show(grid::RegularCartesianGrid{FT, TX, TY, TZ}) where {FT, TX, TY, TZ} =
     "RegularCartesianGrid{$FT, $TX, $TY, $TZ}(Nx=$(grid.Nx), Ny=$(grid.Ny), Nz=$(grid.Nz))"
 
-function domain_string(grid)
+function domain_string(grid::RegularCartesianGrid)
```

Comment:
Again, just pointing out where we have Cartesian, for future changes

Reply:
We'll have to search/replace the whole repo to change `RegularCartesianGrid` to `RegularRectilinearGrid`